frontend/app/models/user.js

```diff
@@ -30,7 +30,7 @@ export default DS.Model.extend(ModelTruncatedDetails, {
   }.property('avatarTemplate'),
 
   coverImageStyle: function() {
```

Comment:
Holy shit how many times is this function repeated I've read this same thing at least 6 times at this point